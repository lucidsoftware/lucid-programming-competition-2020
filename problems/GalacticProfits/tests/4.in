3
3
BoringStock1
1 1 1
BoringStock2
100 100 100
BoringStock3
100 100 100
