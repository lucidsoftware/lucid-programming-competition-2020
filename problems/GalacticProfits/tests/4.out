Max: 0
Min: 0
